git-repository/examples/init-non-bare-repo.rs

```diff
@@ -0,0 +1,33 @@
+// cargo run -p git-repository --example new
```

Comment:
maybe can remove this line -- I always forget the syntax, but might be better in the README.md 

Note: I learned from Rustacean podcast that git-repository was a good starting point, would be helpful if that were near the top of the README (maybe along with some supported use cases, maybe just point to this example folder 😄 )

Reply:
The README can use an overhaul for sure, it's full of information, but it's more like a check-list of what's available than a way to learn where to start.

`gitoxide` is for application developers, for users of its CLI, and sooooon, for the server side. Application developers are completely neglected in the README right now and maybe this is something you could change by adding an MVP-like section that would help you get started with `git-repository`. I'd appreciate it for sure!